themes/default/css/local.css

```diff
@@ -20,13 +555,11 @@
   .sf-menu a:hover {
     text-decoration: underline; }
 
-.sf-menu li:hover ul,
-.sf-menu li.sfHover ul {
+.sf-menu li:hover ul, .sf-menu li.sfHover ul {
```

Comment:
I guess this formatting difference is either from a different version of `scss-cli`, or from passing it different options.